zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AdminRequest.java

```diff
@@ -80,7 +89,8 @@ public void setKey(final long key) {
     this.key = key;
   }
 
-  public void setType(final AdminRequestType type) {
-    this.type = type;
+  public void setConfiguration(final String configuration) {
+    this.configuration = configuration;
+    payloadLength = configuration.length();
```

Comment:
:wrench: This should probably be `configuration.getBytes().lenght()`. String length != number of bytes, depending on the encoding.